performant-software/basira
# BASIRA #113 - Validate image MIME type

**Note**: this PR has a [dependent PR](https://github.com/performant-software/react-components/pull/99) in the react-components repo. Update: Merged 👍 

### What this PR does

- Per #113: 
  - Rejects file uploads that don't match JPG file type and presents an error message

### Status


### Steps to test

1. Visit https://basiraproject-staging.herokuapp.com/
2. Login to the admin interface
3. Edit or create an Artwork
4. Click the "Images" tab
5. Try to upload a non-JPEG file and verify that you get an error message
6. Try to upload a JPEG file and verify that it works
